### ISSUE TITLE
Text in Installer is unreadable dark when Windows Dark Mode is used while HDR is enabled

### ISSUE THREAD

**Issue by BlankFX1** [Rectify11/Installer#345]
### Description

Started the installer for the first time, but the text in the Installer is unreadable dark when Windows Dark Mode is used while HDR is enabled.
Taking a screenshot of it will make the text in the screenshot white and readable. Moving the window to another non-HDR-Monitor does fix it too.
That's all I can say.

### Steps To Reproduce

Just start the installer on an HDR-enabled monitor while windows dark mode is enabled.

### Rectify11 Version

3.0.48.0

### Windows Version

22H2 22621.2428

### Logs

nah

**Comment by BlankFX1** [comment]
Found it that, once installed with dark theme (and mica) this also gets is the case for at least every explorer-window. Literaly all texts except for the ribbon area in the top gets unreadable on HDR Monitors but become readable when drawn to an non-HDR Monitor.
Appears to be a quite critical bug to me.

**Comment by EmberXVII** [comment]
Yeah same issue, on HDR display text is unreadable on the installer and UI within windows when installed.

**Comment by cross** [comment]
Yup.  This makes rectify11 not usable for me.  (I see this in the installer, and based on the above didn't go further.)

**Comment by julianq** [comment]
> Yup. This makes rectify11 not usable for me. (I see this in the installer, and based on the above didn't go further.)

Second this. Found this thread when searching for unreadable text in installer. I won't be able to install until this is fixed.

**Comment by hamburgerghini1** [comment]
It's weird that I have this issue on my desktop but not on my laptop

**Comment by Stryxus** [comment]
This is affecting me on everything when I toggle HDR on. Sometimes the text doesnt render at all.

**Comment by artistro08** [comment]
This affects me when I plug in my laptop into my USB thunderbolt display. No problems when I disconnect it and use the built in screen.

**Comment by Flaflo** [comment]
Can confirm, same thing here with my HDR screen.

**Comment by korolek13** [comment]
Yeah, same for me.

**Comment by M7mdxZ** [comment]
> This affects me when I plug in my laptop into my USB thunderbolt display. No problems when I disconnect it and use the built in screen.

Probably because when it is unplugged, it turns HDR off.

**Comment by Girofox** [comment]
I suspect this issue is more Nvidia related and is appearing in other programs too, for example SystemInformer. I have no Intel Integrated Graphics with HDR support to test though.

**Comment by RTFTR** [comment]
You guys will have to post this issue in MicaForEveryone repo

Since it's the app that fails to render the texts when using it with Mica theme.

**Comment by appl-unique** [comment]
Happens on my non-HDR capable monitor..text is transparent...visible when snipping tool is on...also turns my explorer black

**Comment by appl-unique** [comment]
turns out it was Settings>System>Display>Color management>Automatically manage color for apps....it was on......and i turned it off

**Comment by falcon470** [comment]
Weird that im not seeing thaat option to turn off color management, the toggles just not there

**Comment by falcon470** [comment]
Well i had downloaded windows hdr calibration in my case, uninstalled that now text is normal im not seeing that option in advanced settings mabey they removed it

**Comment by falcon470** [comment]
nm ill try to find out how to get that option to show up, not working thoguht it was toggled hdr off

**Comment by Sombrax25** [comment]
Has there been an update to get HDR working with Mica or is this a dead end?
